src/app/profile/page.tsx

```diff
@@ -40,6 +40,8 @@ export default function Page() {
       await updateProfile(mockProfile.user_id, {
```

Comment:
github is not letting me comment on the actual line lol but can we comment out line 38, i.e.

`await insertProfile(mockProfile);`

 as part of this PR?

cc @jinkang-0 on [this thread](https://calblueprint.slack.com/archives/C05SMFZ45FF/p1697267263063599)

Reply:
yes ma'am